packages/solana/lib/src/rpc/json_rpc_client.dart

```diff
@@ -71,9 +71,7 @@ class JsonRpcClient {
       body: body,
     ).timeout(
       _timeout,
-      onTimeout: () {
-        throw TimeoutException('request timed out');
-      },
+      onTimeout: () => throw TimeoutException('Time out. Request: $body'),
```

Comment:
I wouldn't put the whole body in exception message. For me, it makes more sense to create a custom exception class, e.g.:

```dart
class RpcTimeoutException extends TimeoutException {
  RpcTimeoutException(String url, this.body) : super('Request timed out: $url');

  final dynamic body;
}
```

and then just throw it with

```dart
throw RpcTimeoutException(_url, body);
```

Reply:
Updated.